### PR TITLE
CML 0.3.0 docs update

### PR DESCRIPTION
**Pull request by elleobrien**
For the upcoming CML 0.3.0 docs, we're going to need to add a few details to the docs on dvc.org/doc. Any edits are welcome.

### CODE REVIEW

**Comment by shcheklein** [comment]
@DavidGOrtega @dmpetrov are we merging this?

**Comment by DavidGOrtega** [comment]
@shcheklein We are merging it on Wed 6AM PST

**Review by DavidGOrtega — APPROVED** [review]

File: content/docs/cml/cml-with-dvc.md [hunk 1]
```diff
@@ -135,3 +135,59 @@ env:
...
 </details>
+
+## For GitHub Actions Users: Try the `setup-dvc` Action!
+
+The [iterative/setup-dvc](https://github.com/iterative/setup-dvc) action is a
+JavaScript action that sets up [DVC](https://dvc.org/) in your workflow.
+
+### Usage
+
+This action can be run on `ubuntu-latest`, `macos-latest`, `windows-latest`.
```
Comment (0x2b3bfa0) [inline content/docs/cml/cml-with-dvc.md:146]:
```suggestion
This action can be run on `ubuntu-latest`, `macos-latest` and `windows-latest`.
```
Wouldn't this list be more human-friendly with the `and` conjunction?

**Review by pared — APPROVED** [review]

**Comment by efiop** [comment]
Merging as per request from @DavidGOrtega

**Pull request merged**